rp235x-hal-examples/README.md

```diff
@@ -56,44 +56,119 @@ https://github.com/rp-rs/rp-hal-boards/ for more details.
+   Compiling nb v1.1.0
+   Compiling byteorder v1.5.0
+   Compiling stable_deref_trait v1.2.0
+..
+   Compiling futures v0.3.31
+   Compiling frunk v0.4.3
+   Compiling rp235x-hal v0.2.0
+    Finished `dev` profile [unoptimized + debuginfo] target(s) in 6.23s```
+```
+
+And we see that the RISC-V elf file is now present at `./target/riscv32imac-unknown-none-elf/debug/blinky`:
+```console
+$ file ./target/riscv32imac-unknown-none-elf/debug/blinky
+./target/riscv32imac-unknown-none-elf/debug/blinky: ELF 32-bit LSB executable, UCB RISC-V, RVC, soft-float ABI, version 1 (SYSV), statically linked, with debug_info, not stripped
+```
+
+You can also specify the ARM target directly by using
+`--target thumbv8m.main-none-eabihf` instead of `--target riscv32imac-unknown-none-elf`.
+
+To build, flash and start the application on the RP235x
+you use `cargo run` with one of the following commands:
+```console
+$ cargo run --target thumbv8m.main-none-eabihf --bin blinky
+$ cargo run --target riscv32imac-unknown-none-elf --bin blinky
+```
+
+For a release build pass `--release` to the `cargo build`
+or `cargo run` commands. This will build the example with optimizations enabled:
+```console
+$ cargo run --target thumbv8m.main-none-eabihf --release --bin blinky
+$ cargo run --target riscv32imac-unknown-none-elf --release --bin blinky
+```
 
+For the ARM target all of the examples are built if no `--bin` is specified:
 ```console
-$ cargo run --bin blinky --target=thumbv8m.main-none-eabihf
-   Compiling rp235x-hal v0.10.0 (/home/user/rp-hal/rp235x-hal)
-   Compiling rp235x-hal-examples v0.1.0 (/home/user/rp-hal/rp235x-hal-examples)
-    Finished `dev` profile [unoptimized + debuginfo] target(s) in 1.62s
-     Running `picotool load -u -v -x -t elf target/thumbv8m.main-none-eabihf/debug/blinky`
-Family id 'rp2350-arm-s' can be downloaded in absolute space:
+$ cargo clean
+     Removed 1488 files, 398.6MiB total
+$ cargo build --target thumbv8m.main-none-eabihf
+   Compiling proc-macro2 v1.0.89
+   Compiling unicode-ident v1.0.13
+..
+   Compiling rp235x-hal v0.2.0
+   Compiling pio-proc v0.2.2
+    Finished `dev` profile [unoptimized + debuginfo] target(s) in 16.08s
+$ find target/thumbv8m.main-none-eabihf/debug/ -maxdepth 1 -type f -executable | sort
+target/thumbv8m.main-none-eabihf/debug/adc
+target/thumbv8m.main-none-eabihf/debug/adc_fifo_dma
+..
+target/thumbv8m.main-none-eabihf/debug/blinky
+..
+target/thumbv8m.main-none-eabihf/debug/vector_table
+target/thumbv8m.main-none-eabihf/debug/watchdog
+```
+
+For the RISC-V it is currently possible to build only *some* of the examples. See
+[`riscv_examples.txt`](./riscv_examples.txt) for a list of known working examples.
+The missing ones probably rely on interrupts, or some other thing we
+haven't ported to work in RISC-V mode yet.
+
+Here is an example using the `blinky` example in RISC-V mode. We'll build and
+run it first in non-release emulating the developement cycle **Note:** be sure
```

Comment:
Typo: development

And I think this should talk about compiling using the default debug profile, not running "in non-release"

Reply:
I've used "dev profile" as that seems the correct term from [here](https://doc.rust-lang.org/cargo/reference/profiles.html)